content/en/platform/corda/5.2/developing-applications/api/ledger/utxo-ledger/vault-queries.md

```diff
@@ -240,7 +247,7 @@ class DummyCustomQueryFilter implements VaultNamedQueryStateAndRefFilter<TestUtx
 
 #### Transforming
 
-To create a transformer class, make sure to only keep the transaction IDs of each record. Transformer classes must implement the `VaultNamedQueryStateAndRefTransformer<T, R>` interface. The `<T>` is the type of results returned from the database, which in this case is `TestState`, `<R>` is the type to transform the results into, and transaction IDs should be `Strings`.
+To create a transformer class, only keep the transaction IDs of each record. Transformer classes must implement the `VaultNamedQueryStateAndRefTransformer<T, R>` interface. The `<T>` is the type of results returned from the database, which in this case is `TestState`, `<R>` is the type to transform the results into, and transaction IDs should be `Strings`.
```

Comment:
```suggestion
To create a transformer class, only keep the transaction IDs of each record. Transformer classes must implement the `VaultNamedQueryStateAndRefTransformer<T, R>` interface. The `<T>` is the type of results returned from the database, which in this case is `TestState`, `<R>` is the type to transform the results into, and transaction IDs must be `Strings`.
```
Maybe consider listing T, R and transaction IDs as an unordered list?

Reply:
Updated